doc/source/whatsnew/v0.10.0.rst

```diff
@@ -3,13 +3,6 @@
 v0.10.0 (December 17, 2012)
 ---------------------------
 
-{{ header }}
```

Comment:
do we still need the ``{{ header }}`` ? cc @datapythonista

Reply:
Yes, it should be in all pages. It is where we do the `import pandas as pd` and set some options. But even in pages where there is no code, I need it there because with the new sphinx theme, I'll be controlling the navigation with this.

Reply:
I added it back. It was a mistake. I deleted with the include block.